src/js/chat.js

```diff
@@ -218,23 +221,16 @@ function entityEncode( data ) {
                .replace(/\//g, "&#x2F;")
                // Render a link from anchor tags
                .replace(encodedRegex(/<a .*href=('|\")?(https?:\/\/[^ ]+)\1(?: .*|(?=>))>(.+)<\/a>/gi), function(match, p1, url, contents) {
-                    // Unencode url, reencode it properly, and put it in an anchor
-                    return '<a target="_blank" style="color:#FFF;" href="' + unEntityEncode( url ).replace(/[\W-]/g, function(match) {
-                        return "&#x" + match.charCodeAt(0).toString(16) + ";";
-                    }) + '">' +  contents + '</a>';
-               })
-               // Render a link from anything starting with http/https/www.
-               .replace(encodedRegex(/(?:(https?:\/\/www\.)|(https?:\/\/)|(www\.))([^ ]+)/g), function(match, h1, h2, h3, url) {
-                    // Unencode url, reencode it properly, and put it in an anchor
-                    return '<a target="_blank" style="color:#FFF;" href="' + unEntityEncode( (h1||h2||"http://"+h3) + url ).replace(/[\W-]/g, function(match) {
-                        return "&#x" + match.charCodeAt(0).toString(16) + ";";
-                    }) + '">' +  (h1?h1.replace(/&#x2F;/g,"\/"):h2?h2.replace(/&#x2F;/g,"\/"):h3) + url + '</a>';
-               })
+                   // Unencode url, reencode it properly, and put it in an anchor
+                   return '<a target="_blank" style="color:#FFF;" href="' + unEntityEncode( url ).replace(/[\W-]/g, function(match) {
+                       return "&#x" + match.charCodeAt(0).toString(16) + ";";
+                   }) + '">' +  contents + '</a>';
+               }), {attributes:[{name:'target', value:'blank'}, {name:'style', value:'color:#FFF'}]})
                // TODO: Probably remove eventually, only needed for ChatBot to underline links with Flash chat, which is handled automatically in this chat
                .replace(encodedRegex(/<U>(.+)<\/U>/g), '<span style="text-decoration: underline;">$1</span>')
                // TODO: Probably remove eventually, only needed for existing /me (once the Flash app is removed ACTION should be used)
                .replace(encodedRegex(/<I>(.+)<\/I>/g), '<span style="font-style: italic;">$1</span>');
-}
+}               // Render a link from anything starting with http/https/www.
```

Comment:
I forgot to remove/move the comment... Idk how it got there. I wanted to keep the same structure as before for some reason, so I finished the anchorme in the middle. I get what you mean, will change that.